packages/perseus/src/__tests__/server-item-renderer.test.tsx

```diff
@@ -529,4 +530,24 @@ describe("server item renderer", () => {
             expect(screen.getByRole("textbox")).toHaveValue("-42");
         });
     });
+
+    describe("content editing", () => {
+        it("should show linting errors", () => {
+            // Arrange and Act
+            renderQuestion(itemWithLintingError, undefined, {
+                linterContext: {
+                    contentType: "exercise",
+                    highlightLint: true,
+                    paths: [],
+                    stack: [],
+                },
+            });
+
+            // Assert
+            expect(screen.getByRole("link")).toHaveAttribute(
```

Comment:
I don't think there's a cleaner way to assert this. The link doesn't have text or alt text.

Reply:
Would this work? 

```js
expect(
    await screen.findByText(/Don't use level-1 headings:/),
).toBeVisible();
```

Reply:
Oh, for some reason the DOM around the text was greyed out and I didn't think to dig into it. Yep, there is text I can use. Changed 👍